### DISCUSSION TITLE
OTP

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [mantinedev/mantine#2871] [Feature requests]
Hello,

I hope you're well and everything is good for you.

A fully customizable, one-time password input component for the Mantine.dev.

<OtpInput
value={this.state.otp}
onChange={this.handleChange}
numInputs={6}
separator={<span>-</span>}
size="sm"
type= number | string
placeholder="****"
isDisabled
isInputNum
isInputSecure
>

![1](https://user-images.githubusercontent.com/44679555/199699394-1e5f7aec-b8a4-4dd7-b800-1ccaaabe00d5.jpg)
![2](https://user-images.githubusercontent.com/44679555/199699389-63d80335-5895-4342-9f32-acb4bccb3359.jpg)

Thank you very much.
Best regards,

**Comment by zhulien-ivanov** [comment]
Checkout `PinInput` in https://github.com/mantinedev/mantine/tree/master/src/mantine-labs. It's not officially supported at the moment but it should get there at some point, I guess.

**Reply by YunusEmreAlps** [reply]
Thank You @zhulien-ivanov
